docs/visual-basic/programming-guide/language-features/procedures/how-to-write-an-extension-method.md

```diff
@@ -21,11 +21,13 @@ Extension methods enable you to add methods to an existing class. The extension
     Imports System.Runtime.CompilerServices
     ```
 
-3. Within a module in your new or existing application, begin the method definition with the extension attribute:
+3. Within a module in your new or existing application, begin the method definition with the [Extension](xref:System.Runtime.CompilerServices.ExtensionAttribute) attribute:
```

Comment:
I'm not sure if xref links are translated when docs not in en-us or not. If it will be translated, I think it should be put between back ticks.